litestar/contrib/opentelemetry/config.py

```diff
@@ -77,6 +77,8 @@ class OpenTelemetryConfig:
     OpenTelemetry supports excluding urls by passing an env in the format '{exclude_urls_env_key}_EXCLUDED_URLS'. With
     the default being ``LITESTAR_EXCLUDED_URLS``.
     """
+    exclude_spans: list[Literal["receive", "send"]] | None = None
```

Comment:
I think it would make sense to not restrict this to ASGI spans, as it's global config for the integration that can be used to exclude other spans as well

Reply:
Huh, I am not sure how to make this apply to non ASGI spans, i just copy pasted the docstring from the OTEL docs

https://opentelemetry-python-contrib.readthedocs.io/en/latest/instrumentation/asgi/asgi.html#opentelemetry.instrumentation.asgi.OpenTelemetryMiddleware

I think by default it's just send receive ASGI spans.

Would it make more sense to call it something different on the litestar side

```exclude_asgi_spans```

Reply:
We can even do it similar to how logfire does it for FastAPI `record_send_receive` and default it to False 

https://logfire.pydantic.dev/docs/reference/api/logfire/#logfire.Logfire.instrument_fastapi(request_attributes_mapper)

Reply:
Ah, guess I'm mistaken then. Alright, let's keep this as-is